src/server.js

```diff
@@ -0,0 +1,150 @@
+"use strict";
+var http = require('http');
+var graphql = require('graphql');
+require('babel-register');
+var WebSocketServer = require('websocket').server;
+var graphql_tools = require('graphql-tools');
+var graphql_validator = require('graphql/validation');
+var graphql_execution = require('graphql/execution');
+
+class Server {
+  /*
+  options {
+    schema: GraphQLSchema
+    contextValue?: any,
+    rootValue?: any,
```

Comment:
We'll need to provide contextValue and rootValue per connection. Probably also some kind of functionality for logging in and logging out.